src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -150,18 +164,28 @@ public function execute(WorkflowInstance $workflow)
                 $email = Email::create();
                 try {
                     $email->setTo($member->Email);
-                } catch (Swift_RfcComplianceException $exception) {
+                } catch (RfcComplianceException) {
                     // If the email address isn't valid we should skip it rather than break
                     // the rest of the processing
                     continue;
                 }
                 $email->setSubject($subject);
-                $email->setFrom($from);
+                if ($from) {
+                    $email->setFrom($from);
+                }
```

Comment:
This change fixes https://github.com/silverstripe/.github/issues/394 and will fall back on the default `admin_email` configuration.